### DISCUSSION TITLE
ForceAtlas layout for networkx?

### DISCUSSION THREAD

**Discussion by jonjoncardoso** [networkx/networkx#5205] [Ideas]
Hi, 

I was wondering if there was ever a discussion on whether to include [Force Atlas 2](https://gephi.wordpress.com/2011/06/06/forceatlas2-the-new-version-of-our-home-brew-layout/) layout algorithm in networkX?

I could potentially submit a PR with this feature in the future, but I would like to know if there was any history of discussion for this -- I couldn't find any

**Comment by dschult** [comment]
First, I guess it is relevant to say that the default layout (spring_layout) is a force layout algorithm, though it is not Force Atlas 2.  Take a look at https://github.com/networkx/networkx/pull/2389 for previous discussion that includes a mention of Force Atlas 2. 

In general, we want to include layout algorithms, so it'd be good to have Force Atlas 2 in our library.  We'd certainly appreciate a PR when you get to that point. Also, it's probably best not to wait until it is completely done -- you can submit a "Draft Pull Request" and ask for feedback.
